dev-docs/RFCs/proposals/fixed-frame-coordinates-rfc.md

```diff
@@ -56,20 +61,31 @@ new Layer({
 
 Layers can currently specify coordinates in `COORDINATE_SYSTEM.LNGLAT`, or `COORDINATE_SYSTEM.METER_OFFSETS` (local East-North-Up coordinates from an origin).
 
-Implementation-wise, our layer shaders would use `@math.gl/geospatial` to generate a transform from fixed coordinates to a local ENU coordinate system, multiply that with any model matrix, and then treat the layer data `COORDINATE_SYSTEM.METER_OFFSETS`.
+Implementation-wise, rendering would still happen in "cartographic" our layer shaders would use `@math.gl/geospatial` to generate a transformation matrix from fixed coordinates to a local ENU coordinate system, multiply that with any model matrix, and then treat the layer data `COORDINATE_SYSTEM.METER_OFFSETS`.
```

Comment:
`METER_OFFSETS` only works for local coordinates. How will we handle projection at low zoom levels?

Reply:
Very true, this only handles local scales. For features that are so large that they are visible when zoomed out, we would need full "globe/3D" support (or at least a more complex approach). 

This RFC is just a step on the way towards full 3D support. The point is that with very little effort we can support fixed frame coordinates in many important "local" cases and then we can just extend the support to additional zoom levels when the underlying projection engine supports it.

Will add the caveat.